### DISCUSSION TITLE
Set form validity manually

### DISCUSSION THREAD

**Discussion by jaymehta-twilio** [react-hook-form/react-hook-form#5785] [Q&A] [answered]
Hello,

As shown in the snapshot, we've multiple combo boxes which aren't registered nor controlled within `react-hook-form`. But, there are other controlled input fields within this form only. Is there a way to set validity for the form manually based on the valid scenario for all the combo boxes here. For reference, if none of the boxes have set any value, would be invalid form scenario.

<img width="1442" alt="Screen Shot 2021-06-29 at 5 24 13 PM" src="https://user-images.githubusercontent.com/52717786/123883952-ef1ed480-d8fe-11eb-9c5f-e8508a8b074c.png">

**Comment by bluebill1049** [comment, marked as answer]
https://react-hook-form.com/api/useform/seterror ?

**Reply by jaymehta-twilio** [reply]
Thanks @bluebill1049 . I was under impression that the name that we pass in `setError` is a registered field name. But, it could be any unique string that I can track.
